lib/cisco_node_utils/cmd_ref/vlan.yaml

```diff
@@ -11,6 +11,16 @@ create:
 destroy:
   config_set: "no vlan %s"
 
+mode:
+  _exclude:
+    - /N9K/
+    - /N3/
```

Comment:
Missing 'K' here?  Also, you should be able to exclude these using a single line.

`_exclude: [/N3K/, /N9K/]`